SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -74,32 +72,24 @@ public async Task<IActionResult> UpdateUser(int id, [FromBody] User user)
                 else
                 {
                     User dbUser = null; //assume user does not exist
```

Comment:
Once again, this is not necessary because of the `FirstOrDefault();`

Reply:
Changed